assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -306,6 +306,8 @@ che.infra.kubernetes.namespace.creation_allowed=true
 che.infra.kubernetes.namespace.default=<username>-che
 
 # Defines whether che-server should try to label the workspace namespaces.
+# NOTE: It is strongly recommended to keep this value to true. If false, the new workspace namespaces will not be labeled
+# and therefore not automatically recognized by the Che operator making some features of DevWorkspaces not working.
```

Comment:
should we add a note that technically admin can label namespaces - see https://www.eclipse.org/che/docs/stable/administration-guide/configuring-namespace-strategies/#_labeling_the_namespaces

Reply:
Please take a look at the improved wording.

Reply:
Thanks, could you confirm that the docs are valid and up to date - https://www.eclipse.org/che/docs/next/administration-guide/configuring-namespace-provisioning/

?